students/12th/sangjunkim/login.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Instagram</title>
+    <link rel="stylesheet" href="/styles/login.css" />
+    <script defer src="/js/login.js"></script>
+</head>
+
+<body>
+    <main>
+        <article>
+            <div class="m-left">
+                <img src="https://www.instagram.com/static/images/homepage/home-phones.png/43cc71bb1b43.png" alt="showing how instagram works" />
+            </div>
+            <div class="m-right">
+                <div class="login-info">
+                    <img class="logo" src="/img/logo_text.png" alt="instagram logo" />
+                    <div>
+                        <form class="loginForm" action="">
+                            <div class="login-input">
+                                <input class="idPasswordForm" type="text" name="username"
+                                    placeholder="전화번호, 사용자 이름 또는 이메일" />
+                                <input class="idPasswordForm" type="password" name="password" placeholder="비밀번호" />
+                                <button class="loginBtn">로그인</button>
+                            </div>
+
+                            <div class="divideLogin">
+                               <div class="line"></div>
+                               <div class="lineText">또는</div>
+                               <div class="line"></div>
+                            </div>
+                            <div class="facebookLogin">
+                                <a href="#" >
+                                   <span><img class="FacebookIcon" src="/img/facebook_img.png" alt="facebook icon"></span>
+                                    Facebook으로 로그인
+                                </a>
+                                </div>
+                            <div class="forgotPassword">
+                                <a href="">비밀번호를 잊으셨나요?</a>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+
+                <div class="login-sign-up">
+                    <p>계정이 없으신가요?
+                    <a class="join" href="">가입하기</a>
+                </p>
+                </div>
+                <div class="getApp">
+                    <p>앱을 다운로드하세요.</p>
+                </div>
+                <div class="appBanner">
+                    <a href="#"><img class="appBtn" src="img/appstore_download.png" alt="appstore_download" /></a>
+                    <a href="#"><img class="appBtn" src="/img/googleplay_download.png" alt="googleplay_download" /></a>
+                </div>
+            </div>
+        </article>
+    </main>
+
+    <footer class="footer">
+        <div class="footerBottom">
+            <ul class="footLeft">
+                <li><a href="#">소개</li></a>
```

Comment:
```suggestion
                <li>
                  <a href="#">소개</a>
                </li>
```
- 여는 태그와 닫는 태그가 잘못 설정되어 있네요. 안쪽에 a 태그가 있어야 하고, 그 내용을 li 태그가 감싸도록 해주셔야 합니다.
- 부모 자식 요소가 잘 드러나도록 줄바꿈 해주세요

Reply:
수정하였습니다.